csm/conf/etc/rsyslog.d/0-csm_logs.conf

```diff
@@ -33,8 +33,12 @@ if ($rawmsg contains "csm_agent" ) then {
         action(type="omfile" file="/var/log/seagate/csm/csm_agent.log")
     }
 
+if ($rawmsg contains "cortxcli" ) then {
+        action(type="omfile" file="/var/log/seagate/csm/cortxcli.log")
```

Comment:
Has Log File Name changes been Discussed and approved ??

Reply:
As cli name has changed from csm_cli to cortxcli, So It doesnt make any sense to keep it as old csm_cli.log. Also this cortxcli.log was the part of commit by Eduard at the time of making the changes for sepertae cortxcli rpm. I have just added the log file path to rsyslog config.

Reply:
ohk